### PR TITLE
Add privacy policy email address (Fixes #6820)

### LINKED ISSUE #6820: Update "contact" section of Mozilla Privacy Policy
## Description

Update the copy in https://www.mozilla.org/en-US/privacy/ with the copy and links in [this doc](https://docs.google.com/document/d/19EA0GrF4p0QHqaznkNnuNzm8DyJ18PxBKHgKlewJLf8/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Add email address and Data Subject Access Requests links in contact section of privacy policy.

## Issue / Bugzilla link
#6820

### CODE REVIEW

File: bedrock/privacy/templates/privacy/index.html [hunk 1]
```diff
@@ -48,9 +48,13 @@ <h2>{{ _('Contact Mozilla') }}</h2>
           <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
           <span itemprop="addressLocality">Mountain View</span>,
           <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span>
+          <span itemprop="postalCode">94041</span><br>
         </span>
       </p>
+      <a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a>
+      {% if l10n_has_tag('privacy_email_2019') %}
+      <p>See <a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">here</a> for Data Subject Access Requests.</p>
```
Comment (craigcook) [inline bedrock/privacy/templates/privacy/index.html]: This string needs to be wrapped for l10n, either in a `{% trans %}` block or `_(' ')`

File: bedrock/privacy/templates/privacy/index.html [hunk 2]
```diff
@@ -48,9 +48,13 @@ <h2>{{ _('Contact Mozilla') }}</h2>
           <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
           <span itemprop="addressLocality">Mountain View</span>,
           <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span>
+          <span itemprop="postalCode">94041</span><br>
         </span>
       </p>
+      <a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a>
```
Comment (craigcook) [inline bedrock/privacy/templates/privacy/index.html]: Let's make this part of the address above with `itemprop="email"`

File: bedrock/privacy/templates/privacy/index.html (hunk 1 above)
Comment (amychurchwell) [reply bedrock/privacy/templates/privacy/index.html]: whoops! Thank you!

File: bedrock/privacy/templates/privacy/index.html [hunk 3]
```diff
@@ -48,9 +48,13 @@ <h2>{{ _('Contact Mozilla') }}</h2>
           <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
           <span itemprop="addressLocality">Mountain View</span>,
           <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span>
+          <span itemprop="postalCode">94041</span><br>
+          <span itemprop="email"><a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a></span>
         </span>
       </p>
+      {% if l10n_has_tag('privacy_email_2019') %}
+      {% trans %}<p>See <a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">here</a> for Data Subject Access Requests.</p>{% endtrans %}
```
Comment (craigcook) [inline bedrock/privacy/templates/privacy/index.html]: Actually on second thought, "here" is terrible link text, let's wrap the whole sentence in a link and that also simplifies the whole thing:

```suggestion
      <p><a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">{{ _('See here for Data Subject Access Requests.') }}</a></p>
```

**Comment by craigcook** [comment]
@peiying2 Sorry for pinging after it already merged... the new string is behind the tag `privacy_email_2019`

**Comment by peiying2** [comment]
@craigcook all good. Will get it out soon.

File: bedrock/privacy/templates/privacy/index.html [hunk 4]
```diff
@@ -48,9 +48,13 @@ <h2>{{ _('Contact Mozilla') }}</h2>
           <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
           <span itemprop="addressLocality">Mountain View</span>,
           <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span>
+          <span itemprop="postalCode">94041</span><br>
+          <span itemprop="email"><a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a></span>
         </span>
       </p>
+      {% if l10n_has_tag('privacy_email_2019') %}
+      <p><a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">{{ _('See here for Data Subject Access Requests.') }}</a></p>
```
Comment (peiying2) [inline bedrock/privacy/templates/privacy/index.html]: @amychurchwell should avoid using actual URL in localizable string. Let's replace it with a variable. Filing an issue on this to get this fixed.

Comment (craigcook) [reply bedrock/privacy/templates/privacy/index.html]: The URL is outside the string; the entire string is inside the link: `{{ _('See here for Data Subject Access Requests.') }}`

**Pull request merged**